app/services/idv/inherited_proofing/va/mocks/va_api_client.rb

```diff
@@ -0,0 +1,49 @@
+module Idv::InheritedProofing::Va
+  module Mocks
+    class VaApiClient
+      VALID_AUTH_CODE = 'mocked-auth-code-for-testing'.freeze
+
+      USER_ATTRIBUTES = "{
+        first_name: 'Fakey',
+        last_name: 'Fakerson',
+        address: {
+          street: '123 Fake St',
+          street2: 'Apt 235',
+          city: 'Faketown',
+          state: 'WA',
+          country: nil,
+          zip: '98037'
+        },
+        phone: '2063119187',
+        birth_date: '2022-1-31',
+        ssn: '123456789'
+      }".freeze
```

Comment:
Apparently, the VA decided to return another JSON object, `mhv_data`, that this API returns and we should _probably_ represent it for completeness. You can see the results by running the following from the `identity-idp` repository root:

`$ rails r scripts/inherited_proofing/va/user_attributes/test_script.rb https://staging-api.va.gov private.key`

Where "private.key" should == the name of your private key file located in <identity-idp repro root>tmp/ (it _must_ reside in tmp).

For simplicity, here is the json object:

```
{
  "mhv_data":{
    "mhvId":99999999,
    "identityProofedMethod":"IPA",
    "identityDocumentExist":true,
    "identityProofingDate":"2020-12-14",
    "identityDocumentInfo":{
      "primaryIdentityDocumentNumber":"88888888",
      "primaryIdentityDocumentType":"StateIssuedId",
      "primaryIdentityDocumentCountry":"United States",
      "primaryIdentityDocumentExpirationDate":"2222-03-30"
    }
  }
}
```